notifier.rst

```diff
@@ -133,6 +133,7 @@ Service     Package                          DSN
 ==========  ===============================  ============================================
 Slack       ``symfony/slack-notifier``       ``slack://default/ID``
 Telegram    ``symfony/telegram-notifier``    ``telegram://TOKEN@default?channel=CHAT_ID``
+GoogleChat  ``symfony/googlechat-notifier``  ``googlechat://ACCESS_KEY:ACCESS_TOKEN@default/SPACE?threadKey=THREAD``
```

Comment:
Should be THREAD_KEY instead of THREAD to avoid a mapping (same in the code/recipe)

Reply:
Done